optimade/models/references.py

```diff
@@ -143,7 +143,6 @@ class ReferenceResource(EntryResource):
 
     @validator("attributes")
     def validate_attributes(cls, v):
-        assert any(
-            prop[1] is not None for prop in v
-        ), f"reference object must have at least one field defined"
+        if not any(prop[1] is not None for prop in v):
```

Comment:
@ml-evs Could you affirm this is correct?

Reply:
I'll add a quick test to make sure